package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@mapbox/validator",
+  "version": "0.0.1",
+  "description": "A tiny runtime library for type assertions",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mapbox/validator.git"
+  },
+  "keywords": [
+    "assertion",
+    "type"
+  ],
+  "author": "Mapbox",
+  "contributors": [
+    "David Clark <david.dave.clark@gmail.com>",
+    "Kushan Joshi <0o3ko0@gmail.com>"
+  ],
+  "license": "ISC",
```

Comment:
I've been told that we should leave towards BSD-2 by default. Let's also include a CODE_OF_CONDUCT.md and anything else in https://github.com/mapbox/open-source/issues/12